indexing-service/src/main/java/org/apache/druid/indexing/worker/executor/ExecutorLifecycleConfig.java

```diff
@@ -39,6 +39,10 @@
   @NotNull
   private File statusFile = null;
 
+  @JsonProperty
+  @NotNull
```

Comment:
Isn't it being enforced in ExecutorLifecyle.java?

Reply:
Oh I see, yes you're right. I was trying to find it in this class itself - wondering why the config object itself doesn't verify it instead of the caller.